src/langs/ar.json

```diff
@@ -1,3 +1,28 @@
 {
-  "translations": {}
+  "translations": {
+    "CONTEXT_FETCHING_ERROR_MESSAGE": "حدث خطأ أثناء جلب السياق.",
+    "TOKEN_REQUEST_ERROR_MESSAGE": "حدث خطأ أثناء طلب الرمز المميز.",
+    "CHAT_BOT_ERROR_MESSAGE": "عذرًا، حدث خطأ في برنامج الدردشة الآلية.",
+    "KEYWORD_ALREADY_EXIST_WARNING_MESSAGE": "الكلمة الرئيسية \"{{keyword}}\" موجودة بالفعل. لتحديث تعريفه، قم بحذفه وإدخاله مرة أخرى بالتعريف الجديد.",
+    "FIRST_CHATBOT_MESSAGE": "لقد قمت بالضغط على {{keyword}}، ماذا تريد أن تعرف عنها؟",
+    "MAX_CONVERSATION_LENGTH_ALERT": "لقد وصلت إلى الحد الأقصى لعدد الرسائل المسموح بها في المحادثة",
+    "INPUT_BAR_REPLY_HERE": "الرد هنا...",
+    "INPUT_BAR_SEND_BTN": "يرسل",
+    "BANNER_SHOW_KEYWORDS_BTN": "إظهار الكلمات الرئيسية",
+    "BANNER_HIDE_KEYWORDS_BTN": "إخفاء الكلمات الرئيسية",
+    "KEYWORDS_INFO_ALERT_CHATBOT_ENABLED": "عند تمكين chatbot، يتم تجاهل كافة تعريفات الكلمات الرئيسية. وذلك لأنه يتم عرض chatbot بدلاً من التعريف.",
```

Comment:
I don't know how do you say chatbot in Arabic, but here the IA let it to chatbot.